### ISSUE TITLE
Auxiliary loss is counted twice in the loss

### ISSUE THREAD

**Issue by SizhuangHe** [microsoft/protein-frame-flow#23]
Hi,
In `model_step`, we have `se3_vf_loss += auxiliary_loss`. Yet in `training_step`, we also have `train_loss = (
            total_losses[self._exp_cfg.training.loss]
            +  total_losses['auxiliary_loss']
        )`
The default training loss is `se3_vf_loss` so in this case, auxiliary loss is being counted twice in the loss. Is this expected or a bug?

Thanks!

**Comment by chaitjo** [comment]
I think it may be a bug. We removed and trained and attained very similar results.

**Comment by jasonkyuyim** [comment]
This has been fixed.
